tests/test_workflows.py

```diff
@@ -990,6 +990,13 @@ def test_replace_image(tmp_path):
         i.replace(reader.pages[0].images[0].image)
     assert exc.value.args[0] == "Can not update an inline image"
 
+    import pypdf
+
+    pypdf._page.pil_not_imported = True
```

Comment:
This should use a construct with `finally` to ensure that we always reset the value correctly.